templates/includes/header.html

```diff
@@ -0,0 +1,35 @@
+{% load wagtailcore_tags core_tags static %}
+
+{% secondary_nav_dict as secondary_dict %}
+{{ secondary_dict | json_script:"navigation-data" }}
```

Comment:
I reckon, unless there's a good reason to have them different, that the dict should be used to build the actual menu as well -- That  way the FEDs can see the datastructure, _and_ you can guarantee they're always consistent, rather than maintaining two implementations (it also means fewer function-calls, and/or db queries)

Reply:
As in, one json blob containing primary and secondary nav data? That works for me. (I've already got the mobile nav working locally assuming two blobs, but no biggie to change)

Reply:
More in terms of using `secondary_dict` as the basis for building the HTML menu -- probably should have included more context (joining them into one big blob is also fine by me, but ideally in-addition)

Reply:
agreed I'll go back to this